src/components/Navbar.jsx

```diff
@@ -0,0 +1,134 @@
+                                        <li><Link className="bg-yellow-500 hover:bg-yellow-600 py-2 px-4 block whitespace-no-wrap" to={`/donation`}>School books</Link></li>
+                                        <li><Link className="bg-yellow-500 hover:bg-yellow-600 py-2 px-4 block whitespace-no-wrap" to={`/donation`}>Novels</Link></li>
+                                        <li><Link className="bg-yellow-500 hover:bg-yellow-600 py-2 px-4 block whitespace-no-wrap" to={`/donation`}>Exam Books</Link></li>
+                                    </ul>
+                                </div>
+                                <div className="dropdown inline-block relative">
+                                    <button className="text-white hover:bg-amber-500 hover:text-white px-3 py-2 rounded-md text-sm font-medium py-2 rounded inline-flex items-center">
+                                        <span className="mr-1">Request For</span>
+                                        <svg className="fill-current h-4 w-4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" /> </svg>
+                                    </button>
+                                    <ul className="dropdown-menu absolute z-40 hidden text-white pt-1 flex flex-row">
+                                        <li><Link className="bg-yellow-500 hover:bg-yellow-600 py-2 px-4 block whitespace-no-wrap" to={`/request`}>Books</Link></li>
+                                        <li><Link className="bg-yellow-500 hover:bg-yellow-600 py-2 px-4 block whitespace-no-wrap" to={`/request`}>Stationery</Link></li>
+                                        <li><Link className="bg-yellow-500 hover:bg-yellow-600 py-2 px-4 block whitespace-no-wrap" to={`/request`}>Furniture</Link></li>
+                                        <li><Link className="bg-yellow-500 hover:bg-yellow-600 py-2 px-4 block whitespace-no-wrap" to={`/request`}>School books</Link></li>
+                                        <li><Link className="bg-yellow-500 hover:bg-yellow-600 py-2 px-4 block whitespace-no-wrap" to={`/request`}>Novels</Link></li>
+                                        <li><Link className="bg-yellow-500 hover:bg-yellow-600 py-2 px-4 block whitespace-no-wrap" to={`/request`}>Exam Books</Link></li>
+                                    </ul>
+                                </div>
+                            </>}
+
+                            <Link to="/how-it-works" className="text-white hover:bg-amber-500 hover:text-white px-3 py-2 rounded-md text-sm font-medium">How it works?</Link>
+                            <Link to="/aboutus" className="text-white hover:bg-amber-500 hover:text-white px-3 py-2 rounded-md text-sm font-medium">About Us</Link>
+                            {!isLogin && <Link to="/signup" className="text-white hover:bg-amber-500 hover:text-white px-3 py-2 rounded-md text-sm font-medium">Sign up</Link>}
```

Comment:
Good work with conditional rendering. Please be ready to link this isLogin flag with actual authentication.